python/nwis_client/setup.py

```diff
@@ -28,11 +28,12 @@
 REQUIREMENTS = [
     "pandas",
     "numpy",
-    "requests",
```

Comment:
Is `requests` still required? I had to install it to run.

Reply:
It was for type hints. I removed the import in `iv.py` and changed the type hints to their appropriate type, thus resolving this one. Nice catch, I need to be better about my dev testing environment.